lib/matplotlib/backends/backend_pgf.py

```diff
@@ -94,6 +109,8 @@ def _escape_and_apply_props(s, prop):
     family = prop.get_family()[0]
     if family in families:
         commands.append(families[family])
+    elif not mpl.rcParams["pgf.rcfonts"]:
+        commands.append(r"\rmfamily")
```

Comment:
Is defaulting to serif documented anywhere?  I see one page that says that if `pgf.rcfonts` is disabled, then you will "do the font configuration yourself", which I assumed meant in the preamble.
https://github.com/matplotlib/matplotlib/blob/eb62d6951a5d7657fe006bb919cbae08481023e2/galleries/users_explain/text/pgf.py#L122
For example, will this break expected behavior if a user preamble says `\renewcommand{\familydefault}{\sfdefault}`?

Reply:
I don’t think it’s documented anywhere. I think I made the change to be consistent with the other branches (which always set a font family), and to protect against outer-level font changes messing with the figure fonts when the figure is included in a TeX document. `\fontfamily{\familydefault}\selectfont` may be a better choice?

Reply:
For the moment I changed it to `\fontfamily{\familydefault}`. If anyone else has any opinions about this, they can always speak up ;-)